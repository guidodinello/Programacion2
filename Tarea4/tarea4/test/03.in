# Implementar soloEnA. 
# El iterador 'a' es el parámetro 'it'. 
# El iterador 'b' es el que se lee desde la entrada estándar. 
soloEnA 0
soloEnA 1 4
soloEnA 2 4 8
soloEnA 5 1 3 7 8 9
reiniciar
agregarAIterador 2
soloEnA 0
soloEnA 1 4
soloEnA 2 4 8
soloEnA 5 1 3 7 8 9
soloEnA 1 2
reiniciar
agregarAIterador 4
agregarAIterador 7
agregarAIterador 9
soloEnA 0
soloEnA 1 4
soloEnA 2 4 8
soloEnA 5 1 3 7 8 9
soloEnA 6 1 4 7 8 9 11
Fin
