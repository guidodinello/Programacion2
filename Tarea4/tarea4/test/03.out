1>#  Implementar soloEnA. .
2>#  El iterador 'a' es el parámetro 'it'. .
3>#  El iterador 'b' es el que se lee desde la entrada estándar. .
4>
5>
6>
7>
8>Estructuras reiniciadas.
9>Agregando.
10>2 
11>2 
12>2 
13>2 
14>
15>Estructuras reiniciadas.
16>Agregando.
17>Agregando.
18>Agregando.
19>4 7 9 
20>7 9 
21>7 9 
22>4 
23>
24>Fin.
